src/modules/form/components/DynamicField.tsx

```diff
@@ -218,6 +218,7 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
                 })
```

Comment:
I can't leave a comment on line 204, but I was wondering if you remember @gigxz  the reason for adding `+ 100` here for display types?